### ISSUE TITLE
cargo build fails on i686

### ISSUE THREAD

**Issue by stefson** [CraneStation/cranelift#837]
hey there, not sure wether you're about to have support for bootstrapping to i686 at the moment, but there is a compile failure going on: 

```
   Compiling clap v2.33.0
   Compiling quote v0.6.12
   Compiling term v0.5.2
   Compiling backtrace v0.3.32
   Compiling chrono v0.4.7
error: literal out of range for usize
   --> cranelift-codegen/meta/src/gen_encodings.rs:916:31
    |
916 |             assert!(length <= 0x100000000, "table too big!");
    |                               ^^^^^^^^^^^
    |
    = note: #[deny(overflowing_literals)] on by default
    = note: the literal `0x100000000` (decimal `4294967296`) does not fit into an `usize` and will become `0usize`

error: aborting due to previous error

error: Could not compile `cranelift-codegen-meta`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

this is with rust-1.35.0

**Comment by bnjbvr** [comment]
Thanks for reporting this! I've got a patch.

**Comment by stefson** [comment]
Great, thanks for the fast reply and fix! Do you think it's possible to have a mini-revbump with the fix included? v0.33.1, or the like?

**Comment by sunfishcode** [comment]
@stefson v0.34.0 is now up!
